Sprint1/src/main/java/model/UserManager.java

```diff
@@ -50,4 +45,24 @@ private void LoadFromDisk() {
 			e.printStackTrace();
 		}
 	}
+	
+	public boolean user_input() {
+		boolean logged_in = false;
+		Scanner sc = new Scanner(System.in);
```

Comment:
Since this is a model class we need to move the logging functions into the ConsoleController and ConsoleView. For now this will do but it needs to be refactored before the end of Sprint 1